notebooks/00_quick_start/xdeepfm_movielens.ipynb

```diff
@@ -0,0 +1,762 @@
+{
```

Comment:
<p>it seems like we've integrated the evaluation into the model, curious if we could just get the predictions and then do the same evaluation as in other notebooks using python_evaluation here?</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/643/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
which one did you refer to?
res_syn = model.run_eval(test_file) and res_syn = model.run_eval_topN(test_file,hparams) ?